nixpkgs-src.json

```diff
@@ -1,6 +1,6 @@
 {
   "url": "https://github.com/input-output-hk/nixpkgs",
-  "rev": "6a8a0e57b1ea8bb771a54da66a4b5737de7048c3",
-  "sha256": "1kxkhccvwc3m9wdk9p57bafzgplk68j5l3rh7bkg6hdrzslyw4mh",
+  "rev": "dfdf1a8a5bbad7f21ca23f951e1372d123aedc6e",
```

Comment:
If you create a PR in `input-output-hk/nixpkgs` on `iohk-18.09-angerman` branch I would merge it.

Reply:
I shall do that.